internal/libooniengine/logger.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/ooni/probe-cli/v3/internal/model"
+)
+
+type LogLevel int32
```

Comment:
Cannot these be strings? I think it's easier for a consumer if these are strings.